README.md

```diff
@@ -8,14 +8,14 @@ This library provides some of the new APIs from Scala 2.13 to Scala 2.11 and 2.1
 To use this library, add the following to your build.sbt:
 
 ```
-libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "0.2.2"
+libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "1.0.0"
 ```
 
-Version 0.2.2 is compatible with Scala 2.13.0-M5. For Scala 2.13.0-M4 you should use version 0.1.1.
+Version 1.0.0 is compatible with Scala 2.13.0-RC1 and 2.13.0-M5. For Scala 2.13.0-M4 you should use version 0.1.1. 
 
 Note that there are multiple ways to cross-build projects, see https://github.com/scala/collection-strawman/wiki/FAQ#how-do-i-cross-build-my-project-against-scala-212-and-scala-213.
 
-Also note that this library has not fully stabilized yet. We expect that new, binary incompatible releases of this library will be published (for 2.11, 2.12) until Scala 2.13 is getting close to its final state. Therefore you might want to avoid adding a dependency on that library to your 2.11 / 2.12 artifacts for the time being.
+Starting with version 1.0.0 backwards binary compatibility is enforced within each major version (i.e. anything prior to 2.0.0 will be compatible).
```

Comment:
Should we mention the possibility of a 2.0.0 at all or simply say that all new versions will be backwards compatible? If we get into a situation where we want to break binary compatibility, it's probably best to either resist that urge or publish under a new artifact name. I put the 2.0.0 limit in just in case.

Reply:
I think "within each major version" would be sufficient, but no strong feeling either way.